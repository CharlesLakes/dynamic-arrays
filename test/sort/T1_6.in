10
82 85 35 37 26 78 25 56 60 48