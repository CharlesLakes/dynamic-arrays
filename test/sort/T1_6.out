25 26 35 37 48 56 60 78 82 85